presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueryFramework.java

```diff
@@ -160,13 +160,19 @@ protected void assertQueryFails(@Language("SQL") String sql, @Language("RegExp")
 
     protected void assertQueryFails(Session session, @Language("SQL") String sql, @Language("RegExp") String expectedMessageRegExp)
     {
+        String loggedSql = sql;
+        if (loggedSql.length() > 1000) {
+            //  Avoid too long test output
+            loggedSql = loggedSql.substring(0, 1000) + " ...";
```

Comment:
This isn't needed. It will only be printed if the length check fails. For other test failures, we definitely don't want to truncate the output.